client/js/components/statement/StatementExportModal.vue

```diff
@@ -54,24 +55,45 @@
         <div class="grid grid-cols-5 gap-3 mt-1 mb-5">
           <dp-input
             v-for="(column, key) in docxColumns"
-            :key="key"
             :id="key"
-            type="text"
+            :key="key"
+            v-model="column.title"
             :data-cy="column.dataCy"
             :placeholder="Translator.trans(column.placeholder)"
-            :width="column.width"
-            v-model="column.title" />
+            type="text"
+            :width="column.width" />
         </div>
+        <template v-if="this.active === 'zip' || isSingleStatementExport">
+          <h3
+            id="docxFileName"
+            class="inline-block text-lg mr-1">
+            {{ Translator.trans('docx.export.file_name') }}
+          </h3>
+          <dp-contextual-help
+            aria-labelledby="docxFileName"
+            :text="Translator.trans('docx.export.file_name.hint')" />
+          <dp-input
+            id="fileName"
+            v-model="fileName"
+            :placeholder="Translator.trans('docx.export.file_name.placeholder')"
+            type="text"/>
+          <small>
+            <span
+              class="font-semibold"
```

Comment:
`font-semibold` sadly does not play well with the demosplan-core font stack as some configurations use custom fonts where not all weights are supported. For the time being, use `weight--bold` here.

Reply:
Done in [47a88bc](https://github.com/demos-europe/demosplan-core/pull/3448/commits/47a88bc1302b10013a094a604ce9dd2bd60fed7a)